config/locales/en.yml

```diff
@@ -121,3 +121,28 @@ en:
         roles_and_permissions:        'Users and groups'
         system_status:                'System status'
         technical:                    'Technical'
+  proprietor:
```

Comment:
@mjgiarlo : I have not done this, as I'm having difficulty figuring out a way to run this Hyrax script against Hyku files...it doesn't seem to be included in Hyku (unless I'm missing it).